omnibus/files/private-chef-cookbooks/private-chef/libraries/private_chef.rb

```diff
@@ -178,15 +180,18 @@ def generate_secrets(node_name)
       me = PrivateChef["servers"][node_name]
       ha_guard = PrivateChef['topology'] == 'ha' && !me['bootstrap']
 
+      PrivateChef['postgresql']['db_superuser_password'] ||= generate_hex_if_bootstrap(50, ha_guard)
       PrivateChef['redis_lb']['password'] ||= generate_hex_if_bootstrap(50, ha_guard)
       PrivateChef['rabbitmq']['password'] ||= generate_hex_if_bootstrap(50, ha_guard)
       PrivateChef['rabbitmq']['jobs_password'] ||= generate_hex_if_bootstrap(50, ha_guard)
       PrivateChef['rabbitmq']['actions_password'] ||= generate_hex_if_bootstrap(50, ha_guard)
       PrivateChef['drbd']['shared_secret'] ||= generate_hex_if_bootstrap(30, ha_guard)
       PrivateChef['keepalived']['vrrp_instance_password'] ||= generate_hex_if_bootstrap(50, ha_guard)
+      # In a second or later run, opscode_erchef has become opscode-erchef...
```

Comment:
Can you explain a bit of what you are seeing?  We translate from _ to - as part of a number of services, so my guess is that whatever you are seeing might be broken for other services as well: https://github.com/chef/chef-server/blob/mp/one-connection-type-to-bind-them/omnibus/files/private-chef-cookbooks/private-chef/libraries/private_chef.rb#L288